external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/model_multilabel.py

```diff
@@ -1,4 +1,19 @@
 _base_ = [
   '../../../submodule/models/classification/ote_efficientnet_b0_multilabel.yaml',
 ]
+
+optimizer = dict(lr=0.025)
```

Comment:
Note that changing the LR here normally shouldn't affect the training via OTE CLI or SC, because the value from a model template is suppose to be default in a ote_sdk.entities.train_parameters.TrainParameters instance (so 0.007 will be substituted).

Because each classification template is tied to 3 tasks (multiclass, multilabel, hierarchical), we can't set the optimal learning rate for each task. That's a restriction caused by the current SC design.